pkgs/tools/misc/thinkpad-scripts/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, python3, bash, fetchFromGitHub, python3Packages }:
+
+buildPythonPackage rec {
+  pname = "thinkpad-scripts";
+  version = "4.12.0";
+
+  src = fetchFromGitHub {
+    owner = "martin-ueding";
+    repo = "thinkpad-scripts";
+    rev = "v${version}";
+    sha256 = "08adx8r5pwwazbnfahay42l5f203mmvcn2ipz5hg8myqc9jxm2ky";
+  };
+
+  makeFlags = [
+    "DESTDIR=$(out)"
+    "SPHINXBUILD=${python3Packages.sphinx}/bin/sphinx-build"
```

Comment:
Does the output has to include documentation?

Reply:
It seems to me that yes, unless I patch the Makefile

    common-install:
        install -d "$(DESTDIR)/lib/udev/rules.d/"
        install -m 644 81-thinkpad-dock.rules -t "$(DESTDIR)/lib/udev/rules.d/"
    #
        install -d "$(DESTDIR)/lib/udev/hwdb.d/"
        install -m 644 90-X2x0T-keyboard.hwdb -t "$(DESTDIR)/lib/udev/hwdb.d/"
    #
        install -d "$(DESTDIR)/etc/acpi/events/"
        install -m 644 thinkpad-mutemic-acpi-hook -t "$(DESTDIR)/etc/acpi/events/"
        install -m 644 thinkpad-rotate-acpi-hook-1-normal -t "$(DESTDIR)/etc/acpi/events/"
        install -m 644 thinkpad-rotate-acpi-hook-1-rotated -t "$(DESTDIR)/etc/acpi/events/"
        install -m 644 thinkpad-rotate-acpi-hook-2-normal -t "$(DESTDIR)/etc/acpi/events/"
        install -m 644 thinkpad-rotate-acpi-hook-2-rotated -t "$(DESTDIR)/etc/acpi/events/"
        install -m 644 thinkpad-dock-acpi-hook-1-on -t "$(DESTDIR)/etc/acpi/events/"
        install -m 644 thinkpad-dock-acpi-hook-1-off -t "$(DESTDIR)/etc/acpi/events/"
        install -m 644 thinkpad-dock-acpi-hook-2 -t "$(DESTDIR)/etc/acpi/events/"
    #
        cd desktop && $(MAKE) install
        cd doc && $(MAKE) install
    #

Reply:
You don't have to (though I would have suggested a patch upstream) - you can use split output, see https://nixos.org/nixpkgs/manual/#sec-multiple-outputs- .

Reply:
Or, if you don't mind, you can `rm -r` it in `postInstall`.